src/jshint.js

```diff
@@ -4225,7 +4225,7 @@ var JSHINT = (function() {
     var i = 0;
     var inof = ["in", "of"];
     var level = 0; // BindingPattern "level" --- level 0 === no BindingPattern
-    var comma; // First comma punctuator at level 0
+    var firstcomma; // First comma punctuator at level 0
```

Comment:
That's very strange! Is there any chance you can share a script that demonstrates the crash? I'd like to turn that into a unit test so we can be sure to avoid this problem in the future.